runtime/lua/vim/lsp.lua

```diff
@@ -1313,5 +1313,29 @@ end
 -- Define the LspDiagnostics signs if they're not defined already.
 require('vim.lsp.diagnostic')._define_default_signs_and_highlights()
 
+
+--- Registry for client side code action commands.
```

Comment:
What would be the best way to get this into the lsp.txt docs?

Reply:
it is hard currently :'( we basically only do doc generation for functions. Although my suggestion below would probably make it possible to do so.

Getting closer to getting the tree sitter parser to the point where we could actually do this.